kubernetes/machine-class.yaml

```diff
@@ -42,6 +42,9 @@ providerSpec:
 secretRef: # Secret pointing to a secret which contains the provider secret and cloudconfig
   namespace: default  # Namespace
   name: test-secret # Name of the secret
+# capacityReservation: # Optional - allows targeting of an AWS Capacity reservation or an AWS Resource Group containing reservations. Only one needs to be specified.
+#   capacityReservationId: "cr-05c28b843c05acc11"
+#   capacityReservationResourceGroupArn: "arn:aws:resource-groups:us-west-1:123456789012:group/my-cr-group"
```

Comment:
Hi @shin-nien,

One last change. The sample machine class seems to be wrong here. I think these new fields have to be moved under the `ProviderSpec` [here](https://github.com/gardener/machine-controller-manager-provider-aws/blob/7ca57524d7590afbf1fdfabf72e5205b8db283ec/kubernetes/machine-class.yaml#L12-L41)?

Reply:
Ah yes, of course. Thanks.

Done!